docs/editors-guide/mondo-docs-website-editing.md

```diff
@@ -21,7 +21,7 @@ Get the latest website files locally. This can either be done in the Terminal or
 1. Navigate to the root of the mondo repo, e.g. `cd YOUR-LOCAL-DIRECTORY/mondo`
 1. Fetch the latest content from `master`
 1. Create a feature branch using the GitHub issue number as the branch name, e.g. issue-1234
-1. Run the MkDocs server on your local computer by running this command in your Terminal at the root of the cloned mondo repo (i.e. at the same directory level where the `mkdocs.yml` file is located):  `mkdocs serve`. <br />To stop running the development server type Control+X Control+C in the Terminal.
+1. Run the MkDocs server on your local computer by running this command (`mkdocs serve`) in your Terminal at the root of the cloned mondo repo (i.e. at the same directory level where the `mkdocs.yml` file is located). <br />To stop running the development server type Control+X Control+C in the Terminal.
```

Comment:
@twhetzel were these changes intended on this PR

Reply:
yes